client/src/components/Login.js

```diff
@@ -57,18 +57,57 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('');
+    const [password, setPassword]=useState('');
+    const [errors, setErrors]=useState('');
+
+    const validations = () => {
+        let temp = {};
+        temp.email = email ? "" : "Please enter an email.";
+        temp.password = password ? "" : "Please enter a password.";
+        setErrors({ ...temp });
+
+        return Object.values(temp).every(x => x == "");
+    }
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        if (validations()) {
+            fetch("/auth/login", {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                    'email': email,
+                    'password': password,
+                })})
+                .then(response => response.json())
+                .then(function(response) {
+                    if (response.status === 'success') {
+                        console.log('Success:', email);
+                    }
+                    else {
+                        window.alert(response.message);
+                        console.log(response.message);
```

Comment:
Not necessary for this PR, but eventually we'll want to use something like a snackbar for this kind of notification- https://material-ui.com/components/snackbars/

Reply:
Sounds good!